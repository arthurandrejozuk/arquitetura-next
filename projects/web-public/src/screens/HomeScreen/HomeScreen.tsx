import { Text } from "@alura/design-system/components/Text";
import { sum } from "@alura/utils/math/sum";
import React from "react";

export function HomeScreen() {
    return (
        <div>
            <Text tag={"h1"}>Ol√° mundo</Text>
            <Text tag="p">Importando modulo local: {sum(2, 2)}</Text>
        </div>
    );
}
