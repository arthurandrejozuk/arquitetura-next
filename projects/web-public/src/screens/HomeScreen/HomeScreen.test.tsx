import { HomeScreen } from "./HomeScreen";
import { customRender } from "@alura/test/react.testing-library";

const render = customRender();

describe("<HomeScreen/>", () => {
    it("renders the page", () => {
        const { container } = render(<HomeScreen />);
        expect(container).toMatchSnapshot();
    });
});
