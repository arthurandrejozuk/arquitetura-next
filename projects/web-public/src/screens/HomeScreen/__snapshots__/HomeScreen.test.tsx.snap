// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen/> renders the page 1`] = `
.c0 {
  font-family: sans-serif;
}

<div>
  <div>
    <h1
      class="c0"
    >
      Ol√° mundo
    </h1>
    <p
      class="c0"
    >
      Importando modulo local: 
      4
    </p>
  </div>
</div>
`;
